lib/pages/home/home_controller.dart

```diff
@@ -39,14 +56,41 @@ class HomeController extends GetxController {
     });
   }
 
-  // TODO: update this to get surveys with pagination in [Integrate]
   void _getSurveys() async {
+    if (_paginationFinished) {
+      return;
+    }
+
     final getSurveysUseCase = Get.find<GetSurveysUseCase>();
-    final surveys = await getSurveysUseCase.call("MQ");
-    if (surveys is Success<List<Survey>>) {
-      surveys.value.forEach((element) {
-        print("Items: ${element.title} - ${element.description}");
-      });
+    // ignore: invalid_use_of_protected_member
+    if (_surveys.value.isEmpty) {
+      final surveys = await getSurveysUseCase.call("");
+      if (surveys is Success<List<Survey>>) {
+        surveys.value.forEach((element) {
+          _surveys.add(element);
+        });
```

Comment:
can we use `addAll` instead? 🙏 
```suggestion
        _surveys.addAll(surveys.value);
```